src/routes/Settings.tsx

```diff
@@ -26,6 +26,10 @@ export const APP_VERSION =
       window.electron.packageJson.version
     : 'main'
 
+export const APP_NAME = isDesktop()
```

Comment:
After doing a grep I'm realizing we have a different use of `APP_NAME` in `lib/constants`. Since this is a pretty small PR, could you include a rename of this use of that term to something more specific so there's no confusion between the two?

Reply:
On it

Reply:
What's missing here too in this PR is the fetching of last_download.json so that we can link directly to the right platform and arch. Any suggestion where this async op should occur at?

Reply:
Done. Wanted to also change APP_VERSION to PACKAGE_VERSION but it's actually used in many places so I chickened out haha.

Reply:
Re: fetching of last_download.json: nevermind, I'll create a PR on the website as it's best equipped to do the fetching and parsing. Reverted my commit